src/NewTwitchApi/Cli/CliClient.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace NewTwitchApi\Cli;
+
+use Exception;
+use InvalidArgumentException;
+use NewTwitchApi\Cli\CliEndpoints\CliEndpointInterface;
+use NewTwitchApi\Cli\CliEndpoints\ExitCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetAppAccessTokenCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetGamesCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetStreamsCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetUsersCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetUsersFollowsCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetUserWithAccessTokenCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\GetWebhookSubscriptionsCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\RefreshTokenCliEndpoint;
+use NewTwitchApi\Cli\CliEndpoints\ValidateTokenCliEndpoint;
+use NewTwitchApi\Cli\Exceptions\ExitCliException;
+use NewTwitchApi\Cli\IO\InputOutput;
+use NewTwitchApi\Cli\IO\InputReader;
+use NewTwitchApi\Cli\IO\OutputWriter;
+use NewTwitchApi\Cli\IO\Stdin;
+use NewTwitchApi\HelixGuzzleClient;
+use NewTwitchApi\NewTwitchApi;
+
+class CliClient
```

Comment:
Lot's of effort for CLI interaction. Symfony console component dev dependency won't hurt. Too late to do it now anyway. :)

Reply:
**dev** dependency.